Sources/Networking/OfferingsAPI.swift

```diff
@@ -74,6 +77,27 @@ class OfferingsAPI {
         )
     }
 
+    func getWebBillingProducts(
```

Comment:
I do wonder if this should be out of `OfferingsAPI` since it's not completely related to offerings... We can always move it later though, so I don't think it's a blocker.

Reply:
Done in https://github.com/RevenueCat/purchases-ios/commit/24335549929aeb2cdcd2b4f067734e8077795d39